src/coreclr/vm/ClrEtwAll.man

```diff
@@ -4211,6 +4244,17 @@
                            keywords ="PerfTrackKeyword" opcode="ExecutionCheckpoint" task="ExecutionCheckpoint" symbol="ExecutionCheckpoint"
                            message="$(string.RuntimePublisher.ExecutionCheckpointEventMessage)"/>
 
+                    <!-- Wait Handle events -->
+                    <event value="301" version="0" level="win:Informational" template="WaitHandleWaitStart"
```

Comment:
Maybe `level="win:Verbose"` would be more appropriate for these events, as they can be fairly noisy